cmd/entrypoint/main.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"time"
+
+	"github.com/cybozu-go/github-actions-controller/cmd/entrypoint/cmd"
+)
+
+func main() {
+	cmd.Execute()
+	time.Sleep(time.Duration(1<<63 - 1))
```

Comment:
Let's move this sleep into RunE().

Reply:
Could you make sure that the process terminates when it receives a SIGTERM?

Reply:
Confirmed that the process will terminate when `time.Sleep` receives a `SIGTERM`.
https://github.com/kmdkuk/go-sleep